accelerator/mqtt_interface.c

```diff
@@ -0,0 +1,47 @@
+#include "config.h"
+#include "connectivity/mqtt/client_common.h"
+#include "connectivity/mqtt/duplex_callback.h"
+#include "connectivity/mqtt/duplex_utils.h"
+#include "errors.h"
+
+int main(int argc, char *argv[]) {
+  status_t ret;
+  mosq_config_t cfg;
+  struct mosquitto *mosq = NULL;
+
+  // Initialize `mosq` and `cfg`
+  // if we want to opertate this program under multi-threading, see https://github.com/eclipse/mosquitto/issues/450
+  ret = duplex_config_init(&mosq, &cfg);
+  if (ret != SC_OK) {
```

Comment:
Print error message with logger here.

Reply:
sure of cource. BTW I found the main function of respective protocol (`main.c`, `server.cc`) always return `EXIT_FAILURE` instead of return `status_t`. Which one is better?

Reply:
`EXIT_FAILURE` will shut down TA.
I think we can stick to our original design, use `EXIT_FAILURE` if the initialization failed.